### ISSUE TITLE
Score computed on eval or train mode network?

### ISSUE THREAD

**Issue by lpupp** [BayesWatch/nas-without-training#30]
I'm having a hard time sifting through the code to the find the answer. 

Should one compute the NASWOT score on a network in train or eval mode? 

I get that we are trying to get a measure of how much the network creates these linear separations relative to a dataset. To me this would suggest that something like dropout should not be considered when computing the score and hence would speak towards using an eval model. But including batchnorm seems like it should be since we are normalizing the data (speaking for using the train mode). But if the batch norm is unfit, then possibly eval is better.

input would be appreciated
